src/main/resources/api.yaml

```diff
@@ -301,7 +345,7 @@ paths:
                 $ref: '#/components/schemas/Error'
 components:
   schemas:
-    Account:
+    AccountApi:
```

Comment:
Why need to rename the type? Avoiding naming conflict? 
This is the API spec. Implementation shouldn't affect the name.

Reply:
I changed it due to naming conflict (at least I thought so).

In `AccountApiServiceImpl.java`, line 36:
```
Account account = identity.getAccountService().getAccount(username);
```
The `Account` here should be different from `Account` in OpenApi, which is in the signature of the `accountsPost` method in the same file.